prem/DivAndSpan.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+<head>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>
+        DivAndSpan
+    </title>
+    <style>
+      
+        body {
+            width: 98.5vw;
+            height: 96.5vh;

```

Comment:
Why 98.5vw & 98.5vh?

Reply:
So, i mentioned the width & height of the body in the statement because when i include <!DOCTYPE html> i cant able to get body border instead it shows me a single with 4 px width.
The fact that causes it was the mode which is enabled by the declaration DOCTYPE.

When the DOCTYPE declaration is mentioned, the browser switches to standard mode which requires the height of the document to be mentioned explicitly , so i used those lines to mention height and width of the document.

When the DOCTYPE declaration is not mentioned,  browser works in quirks mode which is compatible mode and renders the page as per author intended. And it implicitly takes the height of the document as auto and set it relative to the viewport.